src/MIDebugEngine/MIDebugEngine.csproj

```diff
@@ -141,6 +141,7 @@
     <Compile Include="Engine.Impl\EngineCallback.cs" />
     <Compile Include="Engine.Impl\EngineUtils.cs" />
     <Compile Include="Engine.Impl\ExceptionManager.cs" />
+    <Compile Include="AD7.Impl\HashAlgorithmId.cs" />
```

Comment:
Can you elaborate? I don't fully understand your comment. 


---
In reply to: [64830281](https://github.com/Microsoft/MIEngine/pull/346#discussion_r64830281) [](ancestors = 64830281)